integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -14,112 +14,188 @@ context('Prisoner Overview page - Education And Training tab', () => {
     cy.task('stubActionPlansList')
     cy.task('getPrisonerById')
     cy.task('getActionPlan')
-  })
-
-  it('should display Education and Training data', () => {
-    // Given
     cy.task('stubLearnerProfile')
     cy.task('stubLearnerEducation')
-
-    cy.signIn()
-    const prisonNumber = 'G6115VJ'
-    cy.visit(`/plan/${prisonNumber}/view/overview`)
-    const overviewPage = Page.verifyOnPage(OverviewPage)
-
-    // When
-    overviewPage.selectTab('Education and training')
-    const educationAndTrainingPage = Page.verifyOnPage(EducationAndTrainingPage)
-
-    // Then
-    educationAndTrainingPage //
-      .activeTabIs('Education and training')
-      .hasFunctionalSkillsDisplayed()
-      .hasCompletedInPrisonQualificationsDisplayed()
+    cy.task('stubGetLongQuestionSetCiagProfile')
   })
 
-  it('should display Education and Training data given curious API returns a 404 for the learner profile', () => {
-    // Given
-    cy.task('stubLearnerProfile404Error')
-    cy.task('stubLearnerEducation')
-
-    cy.signIn()
-    const prisonNumber = 'G6115VJ'
-    cy.visit(`/plan/${prisonNumber}/view/overview`)
-    const overviewPage = Page.verifyOnPage(OverviewPage)
-
-    // When
-    overviewPage.selectTab('Education and training')
-    const educationAndTrainingPage = Page.verifyOnPage(EducationAndTrainingPage)
-
-    // Then
-    educationAndTrainingPage //
-      .activeTabIs('Education and training')
-      .hasFunctionalSkillsDisplayed()
-      .hasCompletedInPrisonQualificationsDisplayed()
+  describe('should retrieve and render data from Curious API data', () => {
```

Comment:
This test file contains the cypress tests for the entire Education and training tab (which contains data pull from both the Curious API, and also the CIAG API)
I have split the tests into 2 `describe` functions - one for Curious API related scenarios and content; and the other for CIAG API related scenarios and content.